src/sonic-config-engine/minigraph.py

```diff
@@ -1685,27 +2168,49 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
 
     if len(system_defaults) > 0:
         results['SYSTEM_DEFAULTS'] = system_defaults
+   
+    if asic_name is not None:
+        results['DEVICE_METADATA']['localhost']['asic_name'] =  asic_name
+    
+    # for single asic Voq Linecards the asic_name needs to populated to "Asic0"
+    if switch_type == "voq" or chassis_type in [CHASSIS_CARD_VOQ]:
+        if not is_multi_asic():
+            results['DEVICE_METADATA']['localhost']['asic_name'] =  "Asic0"
```

Comment:
Does it matter if it is 'Asic0' or' ASIC0' - remember there was an issue earlier

Reply:
This is 'Asic0'